test/test_printBlackjackHand.py

```diff
@@ -0,0 +1,56 @@
+from testing_base import *
+
+def test_printBlackjackHand():
+    shuffled_deck = Deck()
+    shuffled_deck.shuffle()
+
+    Chris = Player('Chris')
```

Comment:
Overall, I think your code looks great, and like it is running which is good! One thing I would want to experiment with is seeing ways we can work it with my code, since it is similar.